### PR TITLE
substitution of abos to dwm

### PR DESCRIPTION
**Pull request by ana-berger**
replicating 'abos' ('dwm' now) in .json files in data_bags/postgresql...

### CODE REVIEW

File: private-sample/data_bags/postgresql_roles/db_po_main.json [hunk 1]
```diff
@@ -770,6 +770,32 @@
...
+        {
+            "name": "dwm_currents",
+            "password": "dwm_currents",
+            "memberof": [
+                "harvest_read_group"
+            ]
+        },
+        {
+            "name": "dwn_ts",
+            "password": "dwm_ts"
```
Comment (mhidas) [inline private-sample/data_bags/postgresql_roles/db_po_main.json:783]: This and all the ones after have a typo. Should be "dwm" not "dwn".

File: private-sample/data_bags/postgresql_databases/harvest.json [hunk 2]
```diff
@@ -1649,6 +1649,90 @@
...
+                    "type": "read"
+                },
+                {
+                    "role": "harvest_write_group",
+                    "type": "write"
+                }
+            ]
+        },
+        {
+            "name": "legacy_dwm",
```
Comment (mhidas) [inline private-sample/data_bags/postgresql_databases/harvest.json:1724]: Don't need "legacy_dwm".

File: private-sample/data_bags/postgresql_roles/db_po_main.json (hunk 1 above)
Comment (ana-berger) [reply private-sample/data_bags/postgresql_roles/db_po_main.json:783]: oh no, silly mistake!

**Comment by mhidas** [comment]
@aodn/devops Along with https://github.com/aodn/chef-private/pull/3583, this is part of https://github.com/aodn/PO-Backlog/issues/1029

From the recent issue we had with the parameters_mapping schema, I gather we have to apply these same changes again in https://github.com/aodn/cloud-deploy/tree/master/sample-config/data_bags for them to appear in the stack db?

**Comment by lwgordonimos** [comment]
@mhidas yes that's correct. In fact, nothing at all uses these postgres databags in chef/private-sample since the old po7 box was deleted. I'm hesitant to delete them here yet because they still form a sample version of the real config which ends up in production and RC, however there is a backlog item to get this messy "3 repo thing" sorted out.

**Comment by mhidas** [comment]
:+1: Thanks @lwgordonimos . Hope that mess can be sorted out soon! Until then...

@ana-berger Please make these same changes in the cloud-deploy repository:
* https://github.com/aodn/cloud-deploy/blob/master/sample-config/data_bags/postgresql_roles/db_po_main.json
* https://github.com/aodn/cloud-deploy/blob/master/sample-config/data_bags/postgresql_databases/harvest.json

**Comment by lwgordonimos** [comment]
@mhidas just confirming this has been done, and commandpost cooked, so the changes should appear in stack databases now.

**Comment by mhidas** [comment]
:+1: Thanks!

**Pull request merged**